optuna/pruners/hyperband.py

```diff
@@ -54,13 +54,21 @@ class HyperbandPruner(BasePruner):
             A parameter for specifying the minimum resource allocated to a trial noted as :math:`r`
             in the paper.
             See the details for :class:`~optuna.pruners.SuccessiveHalvingPruner`.
+        max_resource:
+            A parameter for specifying the maximum resource allocated to a trial noted as :math:`R`
+            in the paper. This value represents and should match the maximum iteration steps (e.g.,
+            ``max_epoch`` for neural networks).
         reduction_factor:
             A parameter for specifying reduction factor of promotable trials noted as
             :math:`\\eta` in the paper. See the details for
             :class:`~optuna.pruners.SuccessiveHalvingPruner`.
         n_brackets:
+
+            .. deprecated:: 1.4.0
+                This argument will be removed from :class:`~optuna.pruners.HyperbandPruner`.
```

Comment:
How about forwarding the reader as follows.

> This argument will be removed from :class:`~optuna.pruners.HyperbandPruner`. The number of brackets are determined based on ``max_resource`` and ``reduction_factor``.

(Note the double backticks around ``max_resource`` and ``reduction_factor``).